src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.tsx

```diff
@@ -39,15 +49,27 @@ const modalStyles = css`
   }
 `;
 
-class EventDetails extends AsyncComponent {
-  static propTypes = {
+type Props = {
+  organization: Organization;
+  location: ReactRouterLocation;
+  eventSlug: string;
+  view: EventView;
+  params: any;
+};
+
+type State = {
+  event: Event;
+};
+
+class EventDetails extends AsyncComponent<Props, State & AsyncComponentState> {
+  static propTypes: any = {
     organization: SentryTypes.Organization.isRequired,
     eventSlug: slugValidator,
     location: PropTypes.object.isRequired,
     view: PropTypes.object.isRequired,
   };
 
-  getEndpoints() {
+  getEndpoints(): Array<[string, string, {query: EventQuery}]> {
```

Comment:
can just be `[string, string, {query: EventQuery}]` right?

Reply:
Nah it cannot. The return type is an array of 3-tuple elements.